presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1836,3 +1836,45 @@ Map of PrestoDB types to the relevant Iceberg types:
 
 
 No other types are supported.
+
+
+Sorted Tables
+^^^^^^^^^^^^^
+
+The Iceberg connector supports the creation of sorted tables.
+Data in the Iceberg table is sorted as each file is written.
+
+Sorted Iceberg tables can provide a huge increase in performance in query times.
```

Comment:
```suggestion
Sorted Iceberg tables can decrease query times in many cases; however, it will depend on the query shape and cluster configuration.
```

Let's not try to make bold claims. I would recommend leaving out the adjective "huge" and try to more objectively describe the feature. Especially when making performance claims, there are many factors which can increase (or decrease) performance. Users will get upset if sorted tables don't _always_ improve performance :)

Reply:
@ZacBlanco It also supports sorting data in Parquet files.

 Data sorted in PARQUET file format
<img width="896" alt="image" src="https://github.com/user-attachments/assets/1e391c05-dad9-4e5f-bf00-5038e646299c" />

Data sorted in ORC file format
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/9723215d-29f3-4b5a-81fb-d3fcb40f9b1a" />

If we set `sorted_writing_enabled` as false and table specifies a sort_by property, it will not throw any error but data in the files are unsorted.

Added test cases for scenarios where sorted writing is disabled, but the table specifies a sort order.

Reply:
> If we set sorted_writing_enabled as false and table specifies a sort_by property, it will not throw any error but data in the files are unsorted.

Is there a reason that this should even be a configuration property and a table property? I don't think it really makes sense to have both. In the case where the configuration is disabled but it is set on the table property, we will potentially get incorrect results for later queries who expect the data in each file to be sorted.